main.py

```diff
@@ -71,6 +72,14 @@ def update(self, case_data):
         self.rpi_array = case_data
         self.last_updated = today
 
+        #Backwards compatability
+        try:
```

Comment:
case_data is the array that comes from the dashboard, so case_data[0] would be the past 24 hours

Reply:
I intentionally wanted to store all the data, including total test and positive tests. The storage impacts should be fairly negligible and would allow looking it all metrics without having to recalculate them later.

Reply:
Ah okay - I'm fine with that too